ports/libgwenhywfar/vcpkg.json

```
@@ -0,0 +1,40 @@
+{
+  "name": "libgwenhywfar",
+  "version-string": "5.6.0",
```

```suggestion
  "version-semver": "5.6.0",
```
It might be more proper to use `version-semver `here. Please see https://github.com/microsoft/vcpkg/blob/master/docs/specifications/versioning.md#22-package-versions